proto/internal/temporal/server/api/persistence/v1/task_queues.proto

```diff
@@ -134,6 +134,13 @@ message TaskQueueTypeUserData {
     DeploymentData deployment_data = 1;
 
     temporal.api.taskqueue.v1.TaskQueueConfig config = 2;
+
+    enum FairnessState {
+        FAIRNESS_STATE_UNSPECIFIED = 0;
+        FAIRNESS_STATE_V1 = 1;
+        FAIRNESS_STATE_V2 = 2;
+    };
```

Comment:
hmm, I think we usually put enums at the top-level. but I guess this is fine, we don't expect to use it anywhere else.